.github/workflows/conda.yml

```
@@ -49,23 +54,34 @@ jobs:
           which python
           pip list
           mamba list
-      - name: Build conda package
+      - name: Build conda packages
         run: |
           # suffix for nightly package versions
           export VERSION_SUFFIX=a`date +%y%m%d`
 
           conda mambabuild continuous_integration/recipe \
+                           --python ${{ matrix.python }} \
+                           --variants "{target_platform: [${{ matrix.arch }}]}" \
+                           --no-test \
                            --no-anaconda-upload \
                            --output-folder .
-      - name: Upload conda package
+      - name: Test conda packages
+        if: matrix.arch == 'linux-64'  # can only test native platform packages
+        run: |
+          conda mambabuild --test ${{ matrix.arch }}/*.tar.bz2
+      - name: Upload conda packages as artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: "conda nightlies (python: ${{ matrix.python }}, arch: ${{ matrix.arch }})"
```

Looks like `:` causes [parsing errors]( https://github.com/dask-contrib/dask-sql/actions/runs/5026700402/jobs/9015287457?pr=1144#step:7:14 )

Maybe this would work?

```suggestion
          name: "conda nightlies (python - ${{ matrix.python }}, arch - ${{ matrix.arch }})"
```